governance/meeting-host-playbook.md

```diff
@@ -0,0 +1,85 @@
+# TAG Environmental Sustainability Meeting Host Playbook
+
+Meetings are held to coordinate and discuss TAG ENV activities. The TAG ENV leads host these meetings. The following playbook can be used for reference to structure the meeting, you don't need to stick step by step (word by word), it should give guidance how a meeting usually is structured.
+
+## Requirements to host the meeting
+
+* Install Zoom and check your settings (especially audio and video).
+* Get access to the [CNCF TAG YouTube Channel](https://www.youtube.com/@CNCFEnvTAG) to start the livestream.
+
+## Pre-Meeting
+* Add a new blank agenda entry to the meeting notes by copying the template and editing the details accordingly.
+* Announce a day in advance in Slack to add agenda items to meeting notes.
```

Comment:
Do we want to include day or week in advance via twitter as well?

Reply:
A day before sounds good.

Reply:
If there is a special session, we might as well announce it a week in advance.